package.json

```diff
@@ -20,14 +20,19 @@
     "test": "gulp test"
   },
   "dependencies": {
-    "appcd-fs": "^1.0.1",
-    "appcd-path": "^1.0.1",
-    "appcd-util": "^1.0.1",
-    "semver": "^5.5.0"
+    "appcd-fs": "^1.1.5",
+    "appcd-path": "^1.1.4",
+    "appcd-util": "^1.1.5",
+    "fs-extra": "^7.0.1",
+    "pluralize": "^7.0.0",
+    "request": "^2.88.0",
```

Comment:
Do we want to be using something like `appcd-request` so that we can respect config settings for proxies? Or is that the point of the `options.network` stuff?